### DISCUSSION TITLE
Predbat addon taking forever to install on vm

### DISCUSSION THREAD

**Discussion by s817itf** [springfall2008/batpred#1760] [General]
I broke my home assistant trying to reboot while installing before, I'm starting on a fresh system, following the youtube videos. Got to the predbat add on install. its just not installing. went and left it for half an hour and its still not completed the install

supervisor log

2024-12-19 11:42:15.039 INFO (MainThread) [supervisor.addons.addon] No 6adb4f0d_predbat addon Docker image 6adb4f0d/amd64-addon-predbat found
2024-12-19 11:42:15.041 INFO (MainThread) [supervisor.docker.addon] Starting build for 6adb4f0d/amd64-addon-predbat:1.2.1

Any help much appreciated

**Comment by gcoan** [comment]
I find it does take quite a while to install, probably 10-15 minutes I think.

It's quite a large add-on, 300Mb or so in size.

Assume you have enough disk space etc?  Anything in the supervisor log ?

**Reply by s817itf** [reply]
I left it for a few hours and it had sorted itself out no idea how long that took as can't see when it fixed itself in the log

**Reply by gcoan** [reply]
Good stuff. Glad to hear you are sorted
